application.py

```diff
@@ -23,6 +29,31 @@
 
 application.logger.info("Flask app created!")
 
+suite = unittest.TestLoader().loadTestsFromModule(test_use_cluster)
+unittest.TextTestRunner(verbosity=2).run(suite)
+
+#For timing purposes
+@application.before_request
+def before_req_func():
+    g.timings = {}
+
+@application.after_request
+def after_req_func(response):
```

Comment:
Hi Jeff, so I was testing the codes a bit, and I found an insight about the timings code. I think we may have to remove it entirely, as it is interfering with the other GET methods in application.py :/ It gives an error with those methods, though it returns the timings with the USE Cluster methods. I think this is only working for POST methods. I have also dropped a message to Turner about this, and thought I could ask him if we should remove the timings code or implement it separately somewhere? After that, I can convert those into logger.info() messages 👍

Reply:
Sure, I think it's fine to remove the timing code for now. We can always grab it from an old commit when we want to use it again! It may be informative to go through the process of learning why it's causing errors in the case of GET requests, but I think it's probably not the most important thing to work on right now.